src/tutorial/apis.md

```diff
@@ -11,7 +11,7 @@ This section shows you how to access Taskcluster APIs programmatically.  The
 examples use JavaScript (ES6) using the JavaScript [taskcluster
 client](https://github.com/taskcluster/taskcluster-client) but apply to [other
 supported
-languages](https://taskcluster-docs.ngrok.io/manual/using/integration/libraries).
+languages](https://docs.taskcluster.net/manual/using/integration/libraries).
```

Comment:
You will find that this redirects to https://docs.taskcluster.net/docs/manual/using/integration/libraries, and in order to make the link work on other deployments, we want to use a `/`-relative link, so `/docs/manual/using/integration/libraries`.